app/src/main/java/org/oppia/app/utility/FontScaleConfigurationUtil.kt

```diff
@@ -0,0 +1,33 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import android.content.Context.WINDOW_SERVICE
+import android.util.DisplayMetrics
+import android.view.WindowManager
+import org.oppia.app.model.StoryTextSize
+
+object FontScaleConfigurationUtil {
+  fun adjustFontSize(
+    context: Context,
+    result: StoryTextSize
+  ) {
+    val configuration = context.resources.configuration
+    configuration.fontScale = getStoryTextSize(result)
+    val metrics: DisplayMetrics = context.getResources().getDisplayMetrics()
+    val wm =
```

Comment:
Prefer ``windowManager`` since it provides context--short names are generally less readable because it's easy to misconstrue their meaning (consider that this code can change overtime and if this variable isn't renamed, it may complicate the understanding of the code over time).

Reply:
Done.